Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM python:3.7.9-slim-buster as base
+
+ENV PATH /usr/local/bin:$PATH
+ENV VIRTUAL_ENV "/venv"
+RUN python -m venv $VIRTUAL_ENV
+ENV PATH "$VIRTUAL_ENV/bin:$PATH"
+WORKDIR /app
+RUN pip install poetry
+EXPOSE 5000
+COPY poetry.lock pyproject.toml /app/
+RUN poetry install --no-root
```

Comment:
Also minor, but this means we're installing dev dependencies for our prod app. This step might be nicer separated between the two stages (using the `--no-dev` flag for prod)